test/ttmlir/Conversion/TosaToTTIR/maximum_op.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: ttmlir-opt --convert-tosa-to-ttir %s | FileCheck %s
+module attributes {} {
+  func.func @test_maximum(%arg0: tensor<13x21x3xf32>, %arg1: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
+    %0 = tosa.maximum %arg0, %arg1 : (tensor<13x21x3xf32>, tensor<13x21x3xf32>) -> tensor<13x21x3xf32>
+    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
```

Comment:
Please add more detailed tests (with detailed regexes), for some examples, take a look at test/ttmlir/Conversion/StableHLOToTTIR/exponential_minus_one_op.mlir

Similar comments for all tests, I'm leaving this on just this one for brevity.